src/endpoints/v1.es6.js

```diff
@@ -686,8 +690,7 @@ class APIv1Endpoint {
           api_type: 'json',
         };
 
-        options.changeSet.forEach((prop) => {
-          options.form[prop] = options.model.get(prop);
+        options.changeSet.forEach((prop) => {options.form[prop] = options.model.get(prop);
```

Comment:
add a newline (formatting), and refactor `=>` into `function` unless you need `this` binding